rst_files/getting_started.rst

```diff
@@ -13,24 +13,16 @@ Overview
 
 In this lecture we will cover how to get up and running with Julia
 
-Topics:
+There are a few different permutations of the setup (cloud-based vs on your machine, native vs :ref:`virtual <jl_jupyterdocker>`, etc.), so some sections may not directly apply to you
 
-#. Jupyter
+That said, you **should still read through them**, as there is useful information throughout
```

Comment:
I don't know about this line -- I think it could be confusing to a lot of people. I think we should just recommend individual users to read through the local installation part

Reply:
Yeah, I am not sure here.  What was the purpose of this line?

Reply:
It was to address the concerns about giving people code to run without explaining it (i.e., we went into detail on the activate_github call in the local install section, and I felt it’d be redundant to say the same stuff under jupyterhub).

I’ve gone and removed this now, at the expense of a little repetition between the cloud and local instructions.